geo-types/src/geometry/geometry_collection.rs

```diff
@@ -248,78 +246,101 @@ impl<'a, T: CoordNum> GeometryCollection<T> {
 }
 
 #[cfg(any(feature = "approx", test))]
```

Comment:
Rather than having several conditional cfgs in the file, I consolidated all the `approx` implementations into one conditional module.